vars.go

```diff
@@ -121,30 +123,52 @@ func AddVulnerability(db *sql.DB, vuln *Vulnerability) error {
 	if rows, _ := res.RowsAffected(); rows < 1 {
 		errs.append(noRowsInserted, "AddVulnerability")
 	}
-	if e := SetCvss(tx, vuln); e != nil {
-		if !IsNoRowsError(e) {
-			return e
+
+	var e interface{}
+	// Setting values in the impact table
+	e = SetImpact(tx, vuln)
+	if ve, ok := e.(Err); ok {
+		if !ve.IsNoRowsError() {
+			return ve
 		}
-		errs.appendFromError(e, "AddVulnerability")
+		errs.appendFromError(ve.err, append([]string{"AddVulnerability"}, ve.parents...)...)
```

Comment:
If you have to do this then you are forcing something that wasn't meant to be. I would suggest looking at the `appendFromError` and follow where it goes. If I remember correctly it checks to see if the error is a `VarsErr`. If it is then it does what you are forcing?